src/sync/syncer2.ts

```diff
@@ -219,7 +221,6 @@ export class OnePubOneWorkspaceSyncer {
         let stats = { ...initialStats };
         let resp : any;
         try {
-            const docs = await this.storage.documents({history: 'all'})
```

Comment:
Turns out this was a bug in my code!  The function accepts a list of docs but doesn't use it, it just queries the storage again.

So I removed this line and we can just use the `docs` that are passed in.

Because of my bug, every time we received a streamed document and tried to pass it along to the other peers, we were actually sending the entire set of documents instead of just the new doc!!  (The call to `_pushDocs` on line 144).  So this fix should make live syncing much more efficient.

Oops :)

Reply:
Let's just tell everyone we've made some new discoveries in syncing science instead 😄